ubi/src/picker.rs

```diff
@@ -269,7 +269,12 @@ impl<'a> AssetPicker<'a> {
             return Ok(matches.remove(0));
         }
 
-        let filtered = self.maybe_filter_for_64_bit_arch(matches);
```

Comment:
Without changing the filter order, "arm" will be excluded because it doesn't contain the string "64".